src/module/get.js

```diff
@@ -0,0 +1,18 @@
+async function getData() {
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/V0CCIRddQzFzK6uRfVbe/scores',
+  );
+  const json = await response.json();
+  return json;
+}
```

Comment:
- [ ] Following the project requirement, it would be better to use arrow functions instead of the `function` keyword.

Reply:
Done.